release.sh

```diff
@@ -23,7 +23,8 @@ get_next(){
 
 get_next_tag(){
   local RELEASE=$1
-  get_next "$( echo "$RELEASE" | jq -r '.tag_name')" "$( git tag | tail -n 1)"
```

Comment:
@tharvik do you understand why this might not work anymore on the latest mac? Symlink from `bash` to `zsh`?

Reply:
indeed, `echo` from zsh differs from the one from bash. in this case, the body of the release contains newlines that get interpreted when echoed. you can prevent it by using `-E`.